techniques/html/H90.html

```diff
@@ -184,26 +181,26 @@ <h2>Related Resources</h2>
 
 
 <section id="related">
-		<h2>Related Techniques</h2>
+		<h2>関連テクニック</h2>
 		<ul>
 			<li><a href="../html/H44">H44: Using label elements to associate text labels with form controls</a></li>
 			<li><a href="../html/H65">H65: Using the title attribute to identify form controls when the label element cannot be used</a></li>
 			<li><a href="../html/H71">H71: Providing a description for groups of form controls using fieldset and legend elements</a></li>
 		</ul>
 	</section>
 <section id="tests">
-		<h2>Tests</h2>
+		<h2>検証</h2>
 		<section class="procedure" id="procedure">
-			<h3>Procedure</h3>
+			<h3>手順</h3>
 			<ol>
-				<li>For each required form control, check that the required status is indicated in the form control's <code class="language-html">label</code> or <code class="language-html">legend</code>.</li>
-				<li>For each indicator of required status that is not provided in text, check that the meaning of the indicator is explained before the form control that uses it.</li>
+				<li>必須項目であるフォームコントロールごとに、そのフォームコントロールの <code class="language-html">label</code> 要素又は <code class="language-html">legend</code> 要素に必須項目であることが示されていることをチェックする。</li>
+				<li>テキストで提供されていない必須項目であることを示すインジケータごとに、そのインジケータを使用しているフォームコントロールの前に意味が説明されていることをチェックする。</li>
```

Comment:
「[テキストで提供されていない必須項目] であることを示すインジケータ」と読めてしまったので、語順を調整してみましたがいかがでしょうか？
```suggestion
				<li>必須項目であることを示す、テキストでは提供されていないインジケータごとに、そのインジケータを使用しているフォームコントロールの前に意味が説明されていることをチェックする。</li>
```